packages/http-client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/http-client",
-  "version": "2.1.1",
+  "version": "3.0.0",
```

Comment:
Why does this need to be a major version bump? Are there any breaking changes here?